src/Persistence/Migrations/20240126104707_ConfigurationModelNameChanged.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Persistence.Migrations
+{
+    /// <inheritdoc />
+    public partial class ConfigurationModelNameChanged : Migration
```

Comment:
Could be. I've advised against deleting migrations since there can be complications. Since one person is on one MR its not an issue, that could change in the future so which could cause problems when deleting migrations.